### ISSUE TITLE
Support PGI C/C++ compilers

### ISSUE THREAD

**Issue by nemequ** [mesonbuild/meson#3231]
Apparently there is support for the PGI Fortran compiler, but not C or C++:

```
CC=pgcc meson build
The Meson build system
Version: 0.44.1
Source dir: /home/nemequ/local/src/uarch-bench/libpfc
Build dir: /home/nemequ/local/src/uarch-bench/libpfc/build
Build type: native build
Project name: libpfc

Meson encountered an error in file meson.build, line 2, column 0:
Unknown compiler(s): [['pgcc']]
```

**Comment by jpakkane** [comment]
Nobody on the development team has access to Portland compilers. This needs to be provided as a MR from somebody who has access to them.

**Comment by nemequ** [comment]
There is a [community edition](http://www.pgroup.com/products/community.htm) which is freely available.

**Comment by scivision** [comment]
This should be closed, was fixed in v0.50
